reference/5.1/Microsoft.PowerShell.Core/About/about_Script_Internationalization.md

```diff
@@ -33,18 +33,18 @@ features:
   information about the Data section, see
   [about_Data_Sections](about_Data_Sections.md).
 
-* New automatic variables, \$PSCulture and \$PSUICulture. \$PSCulture stores the
-  name of the UI language used on the system for elements such as the date,
-  time, and currency. The $PSUICulture variable stores the name of the UI
-  language used on the system for user interface elements such as menus and text
-  strings.
+* New automatic variables, `$PSCulture` and `$PSUICulture`. `$PSCulture` stores
```

Comment:
Some missed `*` to replace with `-` for bullets.

Reply:
Fixed